libs/langchain/langchain/tools/github/tool.py

```diff
@@ -23,11 +23,15 @@ class GitHubAction(BaseTool):
     mode: str
     name: str = ""
     description: str = ""
+    args_schema: Optional[Type[BaseModel]] = None
 
     def _run(
         self,
-        instructions: str,
+        instructions: Optional[str],
```

Comment:
Were you getting back `None` here ever?

Reply:
Yes exactly. Often GPT-4 gives me an `action_input` of `{}` which is parsed into empty `args` being passed into `_run()`, causing it to error out.